packages/wasm-dpp/src/identity/state_transition/identity_topup_transition/identity_topup_transition.rs

```diff
@@ -67,7 +66,7 @@ impl IdentityTopUpTransitionWasm {
     pub fn set_asset_lock_proof(&mut self, asset_lock_proof: JsValue) -> Result<(), JsValue> {
         let asset_lock_proof = create_asset_lock_proof_from_wasm_instance(&asset_lock_proof)?;
 
-        self.0.set_asset_lock_proof(asset_lock_proof);
+        self.0.set_asset_lock_proof(asset_lock_proof).unwrap();
```

Comment:
You need to return error. Try ? or use map_err (see other methods) if it won't work automagically.

Reply:
Agreed